website/docs/tutorials/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,250 @@
+---
+title: Deploying a Java OpenLiberty application with PostgreSQL
+sidebar_position: 1
+---
+
+This tutorial illustrates deploying a [Java OpenLiberty](https://openliberty.io/) application with odo and linking it to an in-cluster PostgreSQL service in the minikube environment.
+
+There are two roles in this example:
+
+1. Cluster Admin - Prepare the cluster by installing the required operators on the cluster.
+2. Application Developer - Imports a Java application, creates a Database instance, and connect the application to the Database instance.
+
+## Cluster admin
+---
+
+This section assumes that you have installed [minikube and configured it](../getting-started/cluster-setup/kubernetes.md).
```

Comment:
there is no content in the page this links to

Reply:
It'll be added as a part of https://github.com/openshift/odo/pull/4973. So we are good to have it, IMO.